### ISSUE TITLE
Question on the argument for backword warping

### ISSUE THREAD

**Issue by suhongkim** [anuragranj/back2future.pytorch#3]
Hi, 

In your Pytorch implementation, I don't understand why you use the forward upsampled flow for backward warping. should it be the backward upsampled flow? (warping from the past feature (feat5c) into the current frame should use the estimated flow ( past->curr) ) 

for example in your code of "back2future.py"
```
        feat5b_warped = self.warp(feat5b, 0.625*flow6_fwd_up)
        feat5c_warped = self.warp(feat5c, -0.625*flow6_fwd_up) # Q: flow6_fwd_up-> flow6_bwd_up?
```

It would be great if you answer my question. 

Thank you, 
Suhong

**Comment by JJanai** [comment]
Hi Suhong,

Thank you very much for pointing out. 
This is actually a small bug in the original code. 
I fixed it in a3b619a it should lead to improvements.

Best,
Joel

**Comment by suhongkim** [comment]
Thank you for a quick reply. 

Best, 
Suhong
